### DISCUSSION TITLE
Create signaling client without this package

### DISCUSSION THREAD

**Discussion by CoreyBovalina** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-js#323] [General]
Is there a way to create a signaling client without this package?  I'm building a react native app that is currently on expo-49, as soon as I update to expo-50 my KVS stream no longer works because this package uses isomorphic-webcrypto.  I basically need a way to create the viewer.signalingclient without this package.  I'm only creating a stream as a viewer.

**Comment by sirknightj** [comment]
Hi @CoreyBovalina,

Yes you can create a signaling client without this package. We have similar implementations of this in [Android](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-android/tree/master/app/src/main/java/com/amazonaws/kinesisvideo/signaling) and [iOS](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios/tree/master/Swift/KVSiOSApp). A wrapper around a WebSocket.

We also have the sigv4 websocket url signer translated into a few languages here: https://github.com/aws-samples/amazon-kinesis-video-streams-demos/tree/master/sigv4-signing
